index.css

```diff
@@ -78,8 +88,372 @@
     margin-left: 10%;
   }
 
+  .works-centered img {
+    width: 100%;
+  }
```

Comment:
- [ ] After adjusting the font size and other elements styles, kindly check that the image is similar to the Figma template otherwise I recommend specifying the width and height.